### PR TITLE
CVL-626 Moving to defining licence policies as code

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/PolicyVersionDifferencesTest.kt [hunk 1]
```diff
@@ -61,6 +62,24 @@ class PolicyVersionDifferencesTest() {
     showDiffs(diffs)
   }
 
+  @Test
+  fun checkCodeVsJson() {
+    assertThat(licencePolicyService.policyByVersion("1.0"))
+      .usingRecursiveComparison()
+      .withStrictTypeChecking()
+      .isEqualTo(POLICY_V1_0)
```
Comment (andrewrlee) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/PolicyVersionDifferencesTest.kt:70]: these tests prove the json and kotlin versions are equivalent and will ensure that the json and kotlin representation stay in sync

Comment (andrewrlee) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/PolicyVersionDifferencesTest.kt:70]: (the recursive comparison bit just gives better feedback of the specific level where difference occur,  the strict typechecking is that it does instance checking too)

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/policies/policyV1.kt [hunk 2]
```diff
@@ -0,0 +1,1506 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
```
Comment (mgreen-mojhub) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/policies/policyV1.kt:1]: Small point, but package could / maybe should be service.policies

Comment (andrewrlee) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/policies/policyV1.kt:1]: yes, totally agree. Is it alright to change in a follow up?

Comment (mgreen-mojhub) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/policies/policyV1.kt:1]: No problem, nothing urgent

**Review by SebNegahbanMoJ — APPROVED** [review]

**Pull request merged**